### ISSUE TITLE
Error with copy-as-markdown on individual records  on debian after results are re-ordered

### ISSUE THREAD

**Issue by alexandrudsc** [dbeaver/dbeaver#17286]
### Description

I have first noticed this on the results of a describe `mysql` statement, I have not tested it on other types of queries (I am not sure if that would make any difference or not).   
    

    

### DBeaver Version

Community Ediotion, Version 21.3.1.202112130652

### Operating System

Debian 11

### Database and driver

1. Database: 10.2.32-MariaDB-log 
2. Driver: org.mariadb.jdbc:mariadb-java-client:RELEASE 2.7.1

### Steps to reproduce

Given a `test` table with 2 columns:
|Field|Type|Null|Key|Default|Extra|
|-----|----|----|---|-------|-----|
|column_1|varchar(100)|YES||||
|column_2|varchar(100)|YES||||

- run an sql statement like
```sql
describe test
```
- right click on the row 2, Advanced Copy -> copy as markdown. The clipboard will contain the markdown for the second row
|Field|Type|Null|Key|Default|Extra|
|-----|----|----|---|-------|-----|
|column_2|varchar(100)|YES||||

- reorder the results based on the column name
![image](https://user-images.githubusercontent.com/7143120/183368068-fa19e609-f2de-48a1-a681-3ed37455c9d0.png)

![image](https://user-images.githubusercontent.com/7143120/183368336-ffc98a3f-31ac-45aa-bbcb-1c202d5d1dcd.png)

- again, right click on the row 2, Advanced Copy -> copy as markdown. The clipboard should contain the markdown for the (new) second row but it contains the markdown for the current first row
|Field|Type|Null|Key|Default|Extra|
|-----|----|----|---|-------|-----|
|column_2|varchar(100)|YES||||

### Additional context

Not sure if this was intended behavior or not. 

OS version
```bash
$ cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

```

Dbeaver version

![image](https://user-images.githubusercontent.com/7143120/183368794-7e8338cf-6d34-4a38-90dc-6c9a1bca6cf5.png)

**Comment by E1izabeth** [comment]
Thank you for bug report. It should be fixed for the DBeaver 22.1.4. Could you, please, check if your issue is still actual for the latest DBeaver version?

**Comment by alexandrudsc** [comment]
Hi @E1izabeth, thanks for your quick reply. Indeed I've updated my dbeaver to 22.1.4 and the issue is fixed.     
    
I will close this then as it is not actual for the latest dbeaver version
